playbooks/installer/ospd/main.yml

```diff
@@ -19,9 +19,17 @@
   hosts: undercloud
   gather_facts: no
   become: yes
-  become_user: "{{ installer.user.name }}"
+  become_method: sudo
```

Comment:
if you have "become: yes" it defaults to "sudo" so no need for the "become_method: sudo" line